lib/index.js

```diff
@@ -31,12 +66,12 @@ export default class Pageres {
 	 * @api public
 	 */
 
-	src(url, sizes, options) {
+	// src(url?: string, sizes?: Array<string>, options?: Options): Pageres | Array<Src> {
+	src(url, sizes, options): Pageres | Array<Src> { // TODO: how can I make all the arguments optional or all required? the commented out line doesn't work as it complains it can also be `undefined`. Can I define a separate type definiton for each scenario?
```

Comment:
@thejameskyle ⬆️ 

`src()` is a getter without arguments and a setter with. Does Flow have a good way of handling this?